requirements/dev-requirements.txt

```diff
@@ -10,15 +10,17 @@ alembic==1.4.3
     # via -r base-requirements.in
 amqp==2.6.1
     # via kombu
+appnope==0.1.2
+    # via ipython
```

Comment:
Do you know why this was added? Doesn't seem like it's necessary in this context (at least).

Reply:
because I made requirements on a mac. ugh